recipes/newrelic/infrastructure/amazonlinux.yml

```diff
@@ -87,25 +87,23 @@ install:
     setup_license:
       cmds:
         - |
-          if [ -f /etc/newrelic-infra.yml ]; then
-            sudo rm /etc/newrelic-infra.yml;
-          fi
-          sudo touch /etc/newrelic-infra.yml;
-        - |
-          sudo tee /etc/newrelic-infra.yml > /dev/null <<"EOT"
+          if [ ! -f /etc/newrelic-infra.yml ]; then
+            sudo touch /etc/newrelic-infra.yml
+
+            sudo tee /etc/newrelic-infra.yml > /dev/null <<"EOT"
           license_key: {{.NEW_RELIC_LICENSE_KEY}}
           enable_process_metrics: true
           EOT
-        - |
-          if [ $(echo {{.NEW_RELIC_REGION}} | grep -i staging | wc -l) -gt 0 ]; then
-            echo 'staging: true' | sudo tee -a /etc/newrelic-infra.yml > /dev/null
+            if [ $(echo {{.NEW_RELIC_REGION}} | grep -i staging | wc -l) -gt 0 ]; then
```

Comment:
@noahmmcgivern we should also have logic to remove the `staging: true` if the current install is not for staging